eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -61,7 +63,7 @@ jobs:
       - ${{ if ne(parameters.testGroup, 'innerloop') }}:
         - '${{ parameters.runtimeFlavor }}_common_test_build_p1_${{ parameters.managedTestBuildOsGroup }}${{ parameters.managedTestBuildOsSubgroup }}_${{ parameters.archType }}_${{parameters.buildConfig }}'
     - ${{ if ne(parameters.stagedBuild, true) }}:
-      - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+      - ${{ format('{0}_{1}_product_build_{2}{3}_{4}_{5}', parameters.runtimeFlavor, parameters.runtimeVariant, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
```

Comment:
If we do reuse runtimeMode here, we probably need some kind of if, because the interpreter does not depend on a special build, while llvm does.

Reply:
That's what I'm wondering. `runtimeVariant` is a compiler flag, `runtimeMode` is a runtime flag.

I guess technically we need both everywhere, since running LLVMJIT Mono without `--llvm` does not in fact use LLVM, so we need a runtime mode passed to a variant runtime, for LLVM, but only a runtime mode passed to a vanilla variant for interpreter 🙃

Reply:
Well, if we are going to have them separate them out that way, we could call it "runtimeBuildVariant" to indicate it is only for builds, and then call the other one "runtimeRunMode". But then you still need logic here to figure out if, based on mode, you need a special build variant here or not. So I am not sure if that simplifies anything?